assignments/assignment-8-rest/assignment-2.html

```diff
@@ -0,0 +1,166 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assigment-8.2</title>
+    </head>
+    <body>
+        <form>
+            <h1>Assignment-2</h1>
+
+            <label for="fullNameInput">Your Full Name</label>
+            <input
+                type="text"
+                placeholder="Your Full Name"
+                id="fullNameInput"
+                name="fullName"
+                required
+            />
+            <label for="emailInput">Your Email</label>
+            <input
+                type="email"
+                placeholder="Your Email"
+                id="emailInput"
+                name="preferredEmail"
+                required
+            />
+            <br />
+            <br />
+            <label for="rangeInput">Rate your Confidence Level</label>
+            <input
+                type="range"
+                id="rangeInput"
+                name="yourConfidenceLevel"
+                min="0"
+                max="10"
+            />
+            <label for="ageInput">Your age</label>
+            <input
+                type="number"
+                id="ageInput"
+                name="age"
+                min="18"
+                max="99"
+                required
+            />
+            <br />
+            <br />
+            <h4>Choose Your Favorite Quiz Categories</h4>
+            <label for="historyInput">
+                <input
+                    type="checkbox"
+                    id="historyInput"
+                    name="quizCategories"
```

Comment:
names should be different for checkbox input in this case. 
isHistoryChecked  isScienceChecked isSportChecked isLiteratureChecked for name attrs